tests/test_commands.py

```diff
@@ -4723,18 +4723,14 @@ def test_interrupted_command(self, r: redis.Redis):
         will leave the socket with un-read response to a previous
         command.
         """
-        print("start")
-
+        
         def helper():
-            try:
+            with pytest.raises(BaseException):
```

Comment:
Note that unless you propagate exceptions from the thread to the test, we'd be unaware of this pytest.raises assertion failing.